test/clj/centrebull/test/db/ranges.clj

```diff
@@ -0,0 +1,8 @@
+(ns centrebull.test.db.ranges
+  (:require [clojure.test :refer :all]))
+
+(defn create!
+  [em rv]
+  (fn [range]
+    (is (= range em))
```

Comment:
It's not a major thing, but it's something I'm working to enforce where I see it. (and trying to use myself).